### ISSUE TITLE
Suggestion: a clearer link for seeing conversation on web page

### ISSUE THREAD

**Issue by jemostrom** [microdotblog/issues#36]
It took me a while before realising that clicking on the timestamp would allow me to see the conversation (in most cases at least). A "conversation" link would make this more obvious.

**Comment by manton** [comment]
Agreed, thank @jemostrom. I think for now I'm going to add a "Conversation" link if it sees that there are replies in the thread. This would also help when viewing the original post, where the timestamp links out to the external site instead of the conversation. (The iPhone app always has a "Conversation" item in the menu already.)

**Comment by ghost** [comment]
This was implemented. Issue no longer needed?

**Comment by issimonwoods** [comment]
The “Conversation” link still only exists when there are replies, so the Conversation view is still hidden until a reply is made.
